src/Faker/Provider/sk_SK/Person.php

```diff
@@ -165,4 +165,46 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+    /**
+     * @example 500620/842
+     *
+     * @link https://en.wikipedia.org/wiki/National_identification_number#Slovakia
+     * @param  \DateTime|null $birthDate date when person to be generated was born
+     * @param  string $gender person gender
+     * @return string generated birth number
+     */
+    public static function birthNumber($birthDate = null, $gender = null)
+    {
+        $birthNumber='';
+        $gender = $gender?$gender:\Faker\Provider\Base::randomElement(array(static::GENDER_MALE, static::GENDER_FEMALE));
```

Comment:
don't make the method `static`, and use `$this->randomElement()` instead.

also, coding standards are wrong, spaces are missing. See the entire codebase for examples.